deployment/legacy-migration/maintenance-worker/migration/neo4j/users/users.cql

```diff
@@ -117,7 +117,7 @@ MERGE (e:EmailAddress {
   verifiedAt: toString(datetime())
 })
 MERGE (e)-[:BELONGS_TO]->(u)
-MERGE (u)<-[:PRIMARY_EMAIL]-(e)
+MERGE (u)-[:PRIMARY_EMAIL]->(e)
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-08-14T12:00:11Z" title="Wednesday, August 14th 2019, 2:00:11 pm +02:00">Aug 14, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -117,7 +117,7 @@ MERGE (e:EmailAddress {
   verifiedAt: toString(datetime())
 })
 MERGE (e)-[:BELONGS_TO]->(u)
-MERGE (u)<-[:PRIMARY_EMAIL]-(e)
+MERGE (u)-[:PRIMARY_EMAIL]->(e)
```


is this code creep?
I realised that this was incorrect during the last deploy to production, but never put in a `PR` to fix it... 
I can separate it into another `PR`, if y'all prefer

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-08-14T15:44:00Z" title="Wednesday, August 14th 2019, 5:44:00 pm +02:00">Aug 14, 2019</time>_

---


Yes, it is unrelated! However it's surprising to me that this was not found earlier.